src/core/Song.cpp

```diff
@@ -457,6 +451,18 @@ void Song::processAutomations(const TrackList &tracklist, MidiTime timeStart, fp
 	}
 }
 
+void Song::setModified(bool value)
+{
+	if( !m_loadingProject )
```

Comment:
Cosmetic, but could be simplified and be more readable.  `value` is ambiguous, something representing the state is a bit more readable.

```cpp
if( !m_loadingProject && m_modified != modified) {
    m_modified = modified;
    emit modified();
}
```